web/app/themes/ppj/src/sass/components/video-player.sass

```diff
@@ -1,4 +1,5 @@
 .video-player
+  $vp: &
```

Comment:
What is the `$vp` variable? I don't understand what it represents or how it's used.

Reply:
I'm using it later in the file here:

  &--playing
    #{$vp}__cover

It's just to stop repetition. In case the name of the block changes, this variable will ensure that the subsequent element selector is named correctly.